content/docs/Other guides/configuring-limits.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Configuring limits"
+linkTitle: "Configuring limits"
+weight: 4
+date: 2020-03-26
+description: >
+  Configure limits to set max number of concurrently running frames
+---
+
+This page describes how to configure limits which allow users to specify 
```

Comment:
Here's a slightly revised version to address a few typos and style issues:

(mostly casing and writing in the present tense)

This page describes how to configure limits that allow users to specify 
the maximum number of concurrently running frames associated with that limit.
Limits are specified on the job layer and all frames within that Layer are 
considered to take 1 limit count. After the total limit count reaches the 
configured max value, the dispatcher stops any additional frames from 
running until a frame with that limit has completed.